components/desktop/mate/mate-session-manager/mate-session-manager.p5m

```diff
@@ -27,17 +27,13 @@ depend type=require fmri=desktop/mate/marco
 
 depend type=require fmri=__TBD pkg.debug.depend.file=usr/bin/dbus-launch
 
-file path=usr/bin/$(MACH64)/mate-session
-file path=usr/bin/$(MACH64)/mate-session-inhibit
-file path=usr/bin/$(MACH64)/mate-session-properties
-file path=usr/bin/$(MACH64)/mate-session-save
-file path=usr/bin/$(MACH64)/mate-wm
 file path=usr/bin/mate-session
 file path=usr/bin/mate-session-inhibit
 file path=usr/bin/mate-session-properties
 file path=usr/bin/mate-session-save
 file path=usr/bin/mate-wm
 file path=usr/share/applications/mate-session-properties.desktop
+#file path=usr/share/doc/mate-session-manager/dbus/mate-session.html
```

Comment:
Why is it commented?

Reply:
It was not delivered so I thought there was a good reason for that. I wanted to make it visible: if things are deliberately not shipped there should be a sign that it was on purpose and not just an overlook.

Reply:
I suppose, in this case it was not on purpose.

Reply:
I can add it then :)